packages/usa-date-picker/src/index.js

```diff
@@ -714,14 +714,14 @@ const disable = (el) => {
 };
 
 /**
- * Check for aria-disabled on initialization
+ * Add the readonly attribute to input element and the aria-disabled attribute to the toggle calendar button and external input elements. 
  *
- * @param {HTMLElement} el An element within the date picker component
+ * @param {HTMLElement} el - The date picker element
  */
 const ariaDisable = (el) => {
   const { externalInputEl, toggleBtnEl } = getDatePickerContext(el);
 
-  el.setAttribute("readonly", "")
+  externalInputEl.setAttribute("readonly", "")
   toggleBtnEl.setAttribute("aria-disabled", true);
   externalInputEl.setAttribute("aria-disabled", true);
```

Comment:
**quibble (non-blocking):** Super minor thing but for consistency, can we group `externalInputEl` changes?

```suggestion
  toggleBtnEl.setAttribute("aria-disabled", true);
  externalInputEl.setAttribute("aria-disabled", true);
  externalInputEl.setAttribute("readonly", "")
```
_Moved line 724 under 726_

Reply:
Resolved in [18e578c](https://github.com/uswds/uswds/pull/6013/commits/18e578c8c71e20ea38cf57edc3e0fdb130e76c0f)